backend/learnings/views.py

```diff
@@ -126,3 +126,24 @@ def create(self, request):
                 is_correct=check,
             )
         return Response({"quiz_taken_id": get_quiz_taken_instance.id})
+
+
+class GetResults(APIView):
+    permission_classes = [IsAuthenticated]
+    authentication_classes = [TokenAuthentication]
+
+    def get(self, request, quiz_taken_id):
+        results = []
+        get_quiz_taken_instance = QuizTaken.objects.get(id=quiz_taken_id)
+        get_user_answers = UserAnswer.objects.filter(quiz_taken_id=get_quiz_taken_instance.id)
+        for data in get_user_answers:
+            get_word = Word.objects.get(id=data.word_id.id)
+            results.append(
```

Comment:
Forgot to this in previous prs, but please refactor this to look like this view `DisplayCategoryWithUserFilter` :bow: 
No need to refactor the ones that I approved earlier